services/buildslave/fabfile.py

```diff
@@ -27,6 +27,7 @@ def task_install(self):
                 self.binDir,
                 mirror_local_mode=True)
             execute(self.update)
+        run('apt-get install libsqlite3-dev')
```

Comment:
This should go in the Ansible config instead: https://github.com/twisted-infra/braid/blob/master/provisioning/roles/requirements-debian/tasks/main.yml#L7